include/zephyr/net/wifi_mgmt.h

```diff
@@ -274,11 +278,23 @@ enum wifi_twt_sleep_state {
 	WIFI_TWT_STATE_AWAKE = 1,
 };
 
+#ifdef CONFIG_WIFI_FEAT_RAW_SCAN_RESULTS
+struct wifi_raw_scan_result {
+	int8_t rssi;
+	int frame_length;
+	unsigned short frequency;
+	uint8_t data[0];
```

Comment:
This can also be fixed: `CONFIG_WIFI_RAW_SCAN_RESULT_LENGTH`

Reply:
This is as per suggestion given by @udaynordic. 
Currently in the event, the memory will be allocated as per the frame length and passed.
instead of passing CONFIG_WIFI_RAW_SCAN_RESULT_LENGTH size data.